src/languages/python.js

```diff
@@ -5,10 +5,14 @@ Website: https://www.python.org
 Category: common
 */
 
-import { UNDERSCORE_IDENT_RE } from '../lib/modes.js';
+import { UNICODE_SUPPORTED, UNDERSCORE_IDENT_RE } from '../lib/modes.js';
```

Comment:
We don't need a check for this (we can assume we only support browsers with unicode regex support), and we don't need the import at all anymore I don't think.
```suggestion
```